layout-management-webapps/src/main/webapp/vue-app/site-navigation/siteNavigationService.js

```diff
@@ -145,3 +145,19 @@ export function getMembershipTypes() {
     }
   });
 }
+
+export function getPageTemplateCategories() {
```

Comment:
can you retrieve directly page templates from the rest service and not page templats categories ?